LayoutTests/svg/zoom/text/zoom-text-and-geometry.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+body {
+    font-size : 16px;
+    font-family : "Times New Roman";
+    zoom: 2;
+}
+rect {
+    fill : green;
+    stroke: green;
+}
+text {
+    fill : red;
+}
+.geometricPrecision {
+    fill : red;
```

Comment:
No need to include `fill : red;` here since it will be applied by the `text` css rule above.